### ISSUE TITLE
Solid variant

### ISSUE THREAD

**Issue by TomJansen** [pwyde/monochrome-kde#1]
Hi! I love this theme. Thank you very much for making it. I have one question though, if it's not too much to ask: can you please make a solid variant for the plasma theme? That would make it perfect.

Thanks again!

**Comment by pwyde** [comment]
Thank you for the kind words. If you are using the Kvantum theme engine as per [instructions](https://github.com/pwyde/monochrome-kde#application-style), then there already is a solid version. Just launch the **Kvantum Manager** > **Change/Delete Theme** > **Select a theme** drop-down > select **MonochromeSolid**.

Are you referring to Plasma panels and widgets on the desktop?

**Comment by TomJansen** [comment]
Yes I am referring to the Plasma panels and widgets. Sorry if that was not clear. I do not use Kvantum... would that fix this issue?

**Comment by pwyde** [comment]
Unfortunately not. Kvantum theme engine is independent from Plasma desktop theme.

I can make a solid version. Makes sense since I have made a solid Kvantum theme. I will add this to the todo list. I cannot give an ETA on when it will be finished. Hope you understand :smiley:.
